### PR TITLE
MODULE 1:  HELLO MICROVERSE PROJECT

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
### In this project: 

>  I created an HTML page with a  **Hello Microverse header text**
>  I styled the text with a blue color.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+header {
+    color: #000fff;
+}
```
Comment (ahmetbozaci) [inline style.css:8]: You should fix the linter errors.

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️ 

## Hello @Ntwali-Josue ✋🏼  Your project is great. You almost done. However there are some issue you should fix them. Please fix them then your project will approve.

### 💯  Nice to have 💯 
- You setup linter files.
- You created html and css files.
- Your pull request is very professional. It has a meaningful title and summary. Keep like that. 👏🏼 


### ♻️ Request changes ♻️ 

- [ ] You should update your README file. It is good but there some broken links and empty sections. If you wont use some section delete it.
- [ ] You have linter errors. Please fix them.


_**Check the comment about that  under the review.**_


## Optional suggestions

#### These are optional but highly recommmended.

- [x] Your commit messages always should start  with capital letters. I see some of them are not like that. You can change them with using `git rebase`. Please watch this [video](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [ ] For more professional readme you should add screenshot of your project in README file.

### Cheers and Happy coding! 💻 👏
cat on laptop
___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

File: style.css (hunk 1 above)
Comment (Ntwali-Josue) [reply style.css]: Linters errors are fixed.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  color: #000fff;
+}
```
Comment (Emmyn5600) [inline style.css:8]: - [ ] Use h1 tag instead of header element in your css.

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @Ntwali-Josue  :raised_hands:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer.
## To highlight:
   * #### You have the descriptive Readme:heavy_check_mark:
   * #### Your commits look more professional:heavy_check_mark:
   * ####  Linters are passing successful:heavy_check_mark:
   * #### Nice to follow Gitflow :heavy_check_mark:


## Optional suggestions

    _Please check the comments under the review too._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**